build.gradle

```diff
@@ -24,23 +24,34 @@ subprojects {
         addons
     }
 
-    // rosuda jars are required for both par-connector and par-script
-    dependencies {
-        addons "org.rosuda:jriengine:0.5.x"
-        addons "org.rosuda:jri:0.5.x"
-        addons "org.rosuda:rengine:1.7.x"
-    }
 }
 
-project(':par-script') {
+def isWindows = System.properties['os.name'].toLowerCase().contains('windows')
+def isMac = System.properties['os.name'].toLowerCase().contains('mac')
```

Comment:
I don't manage to make this thing work, additionnaly it is marked as
"Incubating", meaning that the API can change at any time. I'd prefer to
stick to the olde way

On Wed, Nov 4, 2015 at 4:13 PM, Laurent Pellegrino <notifications@github.com
> wrote:

> In build.gradle
> <https://github.com/ow2-proactive/connector-r/pull/76#discussion_r43891698>
> :
>
> >  }
> >
> > -project(':par-script') {
> > +def isWindows = System.properties['os.name'].toLowerCase().contains('windows')
> > +def isMac = System.properties['os.name'].toLowerCase().contains('mac')
>
> OperatingSystem is a class from Gradle API.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ow2-proactive/connector-r/pull/76/files#r43891698>.
>